Gemfile

```diff
@@ -23,10 +23,16 @@ group :development do
 end
 
 # Required to make imports in Active Admin stylesheet work
-gem 'sassc-rails', '~> 1.0'
+gem 'sassc-rails', '~> 2.1'
```

Comment:
I removed the version here in a recent change since it is already specified in the gemspec. Bundler outputs a warning if you specify the version in both places.

Reply:
I have removed this from Gemfile, why do we need to mention it in the Gemfile?

Reply:
Mentioning it in the Gemfile causes the gem to be required when the dummy application boots. At one point this was needed to keep ActiveAdmin from raising an exception in Pageflow gem tests. Maybe this problem went away?

Reply:
I tested bin/rspec after removing this and executed host application. I did not get any error

Reply:
Then let's remove it.